hs_core/models.py

```diff
@@ -1381,6 +1381,20 @@ def get_crossref_deposit_xml(self, pretty_print=True):
 
         return '<?xml version="1.0" encoding="UTF-8"?>\n' + etree.tostring(ROOT, pretty_print=pretty_print)
 
+    @property
+    def verbose_name(self):
+        from hs_core.hydroshare.utils import get_resource_types
+        res_cls = None
+        for tp in get_resource_types():
+            if self.resource_type == tp.__name__:
+                res_cls = tp
+                break
+        if res_cls is not None:
+            res_obj = res_cls()
+            return res_obj._meta.verbose_name
+        else:
+            return self._meta.verbose_name
```

Comment:
@xuezhaokun @pkdash I think if get_content_model() works, we should go with that rather than the current implementation for both simplicity and performance reasons.